pyiode/pyiode_exec.pyx

```diff
@@ -6,18 +6,109 @@
+#   idt_execute(sample: Optional[Union[str, List[str]]] = None, idt_list: Optional[Union[str, List[str]]] = None, var_files: Optional[Union[str, List[str]]] = None, scl_files: Optional[Union[str, List[str]]] = None,trace: int = 0) | Execute a list of identities on a given sample
 
+cdef extern from "iode.h": 
+    cdef int    IodeExecuteIdts(char *smpl, char *idt_list, char *var_files, char *scl_files, int trace)
 
-def idt_execute(idt_list: Union[str, List[str]], 
-                sample: Union[str, List[str]], 
-                var_files: Union[str, List[str]], 
-                scl_files: Union[str, List[str]]):
-    '''
-    Estimate a (block of) equations
-     
+def idt_execute(sample: Optional[Union[str, List[str]]] = None, 
+                idt_list: Optional[Union[str, List[str]]] = None, 
+                var_files: Optional[Union[str, List[str]]] = None, 
+                scl_files: Optional[Union[str, List[str]]] = None,
+                trace: bool = False):
+    r'''
+    Execute a list of identities
+    
+    Parameters
+    ----------
+    sample: Optional[Union[str, List[str]]] = None
+            range of period on which the identities must be calculated
+            if sample is None or empty, the WS sample is used 
+            
+    idt_list: Optional[Union[str, List[str]]] = None
+            list of identities to execute
+            if empty, all identities are executed
+            
+    var_files: Optional[Union[str, List[str]]] = None
+            list of files the needed variables must be read from. "WS" means current current Variable WS.
+            if empty, only the current KV_WS is used
+            
+    scl_files: Optional[Union[str, List[str]]] = None
+            list of files the needed scalars must be read from. "WS" means current current WS.
+            if empty, only the current workspace is used
+                        
+    trace: int = 0
+            optional trace indicating the source of the variables and scalars. For example:
+            
+            Execution of identities
+            
+            Parameters
+                Execution sample : 1960Y1:2015Y1
+            
+                Variables loaded
+                    From WS : AOUC AOUC_ COTRES DEBT DTF DTH EFMY EFXY EX FLG FLGR GAP2 GAP_ GOSF HF  
+                    
+                Scalars loaded
+                    From ../data/fun : gamma gamma2 gamma3 gamma4 knf3
+            
+    Examples
+    --------
+        # define the "trace" file
+        iode.w_dest("test_iode.htm", iode.W_HTML)
+    
+        # Simple case: no vars in WS, sample given
+        iode.ws_clear_var()
+        iode.ws_sample_set("1960Y1", "2015Y1")
+        iode.ws_load_idt(f"{IODE_DATA_DIR}fun")
+        iode.idt_execute("1961Y1:2000Y1", "AOUC", f"{IODE_DATA_DIR}fun", trace = 1)
```

Comment:
trace = 1 -> trace = True

Please DO NOT use int where you should use a `bool` value in Python.